dv/verilator/pcount/cpp/ibex_pcounts.cc

```diff
@@ -0,0 +1,62 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <numeric>
+#include <sstream>
+#include <string>
+#include <vector>
+
+#include "ibex_pcounts.h"
+
+// see mhpmcounter_incr signals in rtl/ibex_cs_registers.sv for details
+
+const std::vector<std::string> ibex_pcount_names = {
+    "Cycles",
+    "NONE",
+    "Instructions Retired",
+    "LSU Busy",
+    "Fetch Wait",
+    "Loads",
+    "Stores",
+    "Jumps",
+    "Conditional Branches",
+    "Taken Conditional Branches",
+    "Compressed Instructions"};
+
+std::string ibex_pcount_string(uint64_t pcounts[], bool csv) {
+  char seperator = csv ? ',' : ':';
+  unsigned int longest_name_length;
```

Comment:
I'd probably drop the use of `unsigned` here. A number of coding styles are recommending against using `unsigned` unless strictly necessary (see [here](http://lists.llvm.org/pipermail/llvm-dev/2019-June/132890.html) for some background), and obviously you end up needing the cast for `pcount_name.length()` below.